src/scripts/components/structures/MainHeader.js

```diff
@@ -4,17 +4,14 @@ import MainHeaderLogo from '../widgets/MainHeaderLogo'
 import SearchInput from '../widgets/SearchInput'
 import MainNavigation from '../widgets/MainNavigation'
 import { Link } from 'react-router-dom'
-import Blockies from 'react-blockies'
 
 type Props = {
   children: Object
 }
 
```

Comment:
We have two files named `MainHeader.js` that live in different subdirectories. Can we change one of them? I would think one of them must be named incorrectly

Reply:
Whatever you want.
What is this `/components/MainHeader.js`?